src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PokeCard</title>
+</head>
+<body>
+  <div class="modalBackground hidden">
+    <div class="modal">
+      <section class="cardTotal">
+        <div class="cardModal"></div>
+        <canvas id="grafica"></canvas>
+        <button class="close">X</button>
+      </section>
+      <section class="commentsTotal">
+        <div id="comments-container">
+          <div class="messagesAndCount">
+            <h2>Comments </h2>
+            <div class= "messageCount"></div>
+          </div>
+          <div class="comments"></div>
+        </div>
+        <div class="form">
+          <form action="submit">
+            <ul class="ul-form">
+              <input class="name-form"
+              type="text"
+              placeholder="Your name"
+              required>
+              <input class="insights-form"
```

Comment:
- Optional. Consider changing this field to a `textarea`. It would make much more sense since that is what is used to take message inputs in form fields and it would also match the wireframe.